Makefile

```diff
@@ -12,41 +15,11 @@ config:
 	@ echo # Produce a new line at the end of each target to help readability
 
 .PHONY: verify-config
-verify-config: $(GOPATH)/bin/checkconfig
+verify-config:
 	@ $(ECHO) "\033[36mVerifying Config\033[0m"
-	${GOPATH}/bin/checkconfig --config-path=config/config.yaml --job-config-path=config/jobs --plugin-config=config/plugins.yaml
+	docker run --rm -v $(shell pwd)/config:/config gcr.io/k8s-prow/checkconfig:v20200220-18fae0a00 --config-path=/config/config.yaml --job-config-path=/config/jobs --plugin-config=/config/plugins.yaml
 	@ echo # Spacer between output
-	make config
+	make run-config
 	@ $(ECHO) "\033[36mVerifying Git Status\033[0m"
-	@ if [ "$$(git status -s)" != "" ]; then git diff --color; $(ECHO) "\033[31;1mERROR: Git Diff found. Please run \`make config\` and commit the result.\033[0m"; exit 1; else $(ECHO) "\033[32mValid config found\033[0m";fi
-	@ echo # Produce a new line at the end of each target to help readability
-
-$(GOPATH)/bin/checkconfig:
-	@ $(ECHO) "\033[36mInstalling checkconfig\033[0m"
-	mkdir -p $$GOPATH/src/k8s.io
-	# Clone the test-infra source so that we can use the proper go.mod
-	cd $$GOPATH/src/k8s.io; git clone https://github.com/kubernetes/test-infra
-	# This is the same Git SHA that the images we use are tagged with
-	cd $$GOPATH/src/k8s.io/test-infra; git checkout 3b3e19a133321bb9e451f1c7ce11ed8d8561705f
-	cd $$GOPATH/src/k8s.io/test-infra; GOPROXY=https://proxy.golang.org GOSUMDB=sum.golang.org GO111MODULE=on go install k8s.io/test-infra/prow/cmd/checkconfig
-	@ echo # Produce a new line at the end of each target to help readability
-
-.PHONY:
-check-image-tags:
-	@ $(ECHO) "\033[36m\033[1mChecking image tags\033[0m"
-	scripts/check-image-tags.sh
-	@ echo # Produce a new line at the end of each target to help readability
-
-TAG ?= v20190821-328974b
-.PHONY:
-update-image-tags:
-	@ $(ECHO) "\033[36m\033[1mUpdating image tags\033[0m"
-	scripts/update-image-tags.sh $(TAG)
```

Comment:
I would however keep this script as it can be useful if/when we decide to update all image tags.